nservicebus/azure-service-bus/token-provider.md

```diff
@@ -14,7 +14,7 @@ This is useful when, for example, delegating authentication and authorization to
 
 Changing the authentication mechanism is done using the [Azure Service Bus SDK's TokenProvider class](https://msdn.microsoft.com/en-us/library/microsoft.servicebus.tokenprovider.aspx). The Azure Service Bus SDK requires an instance of this class at 2 different levels.
 
- * `NamespaceManager`: requires a `TokenProvider` that issues tokens with manage rights on the namespace. Note that this is only needed if queue creation is enabled, so that it can list, create and update entities in the namespace.
+ * `NamespaceManager`: requires a `TokenProvider` that issues tokens with manage rights on the namespace. Note that this is only needed if [queue creation](/nservicebus/transports/queuecreation) is enabled, so that it can list, create and update entities in the namespace.
```

Comment:
I'm not sure what value that link is providing. The context here is `NamespaceManage` and having managed rights to create queues. Not manually create queues.